geopandas/io/file.py

```diff
@@ -3,6 +3,7 @@
 import fiona
 import numpy as np
 import six
+from shapely.geometry import shape
```

Comment:
Unless I'm missing something, this import doesn't look necessary.

Reply:
yes, thanks for noticing. I had an intermediate version that needed this :-)